web/components/templates/dashboard/dashboardPage.tsx

```diff
@@ -947,6 +958,29 @@ const DashboardPage = (props: DashboardPageProps) => {
                 </StyledAreaChart>
               </div>
 
+              <div key="threats">
+                <StyledAreaChart
+                  title={"Threats"}
+                  value={`${metrics.totalThreats.data?.data?.toFixed(0) ?? 0}`}
+                  isDataOverTimeLoading={overTimeData.ttft.isLoading}
```

Comment:
nit: what does `ttft` stand for? Might be better to name this so that we know what it is in the future.